### DISCUSSION TITLE
Would it be possible to migrate from the old yaml manifest install to a helm install?

### DISCUSSION THREAD

**Discussion by 2fst4u** [rook/rook#11540] [Q&A] [answered]
My cluster was installed with the old method of downloading the example yamls and editing to suit. I would prefer to change to helm to make it easier to upgrade and keep track of only the values I care about rather than a large yaml of lines I have no need to change.

**Comment by travisn** [comment, marked as answer]
@2fst4u Try [this topic](https://rook.io/docs/rook/latest/Helm-Charts/ceph-cluster-chart/#existing-clusters) to add labels to the resources so you can install the helm chart to start managing them.

**Reply by 2fst4u** [reply]
That looks like it should do the trick. Thanks.
